lib/web/integrations.go

```diff
@@ -109,14 +115,25 @@ func (h *Handler) integrationsUpdate(w http.ResponseWriter, r *http.Request, p h
 			return nil, trace.BadParameter("cannot update %q fields for a %q integration", types.IntegrationSubKindAWSOIDC, integration.GetSubKind())
 		}
 
+		issuerS3URI := url.URL{
```

Comment:
Should we validate URL before setting it to the integration? Also, we allow absence of url here?

Reply:
The s3 fields are not required.
As for validating the URL, given that we already validate it in UpdateIntegration grpc call, I think it's fine to only validate the presence of values here and delegate everything else to the UpdateIntegration

Reply:
I've made them required now :+1: